src/actions/index.js

```diff
@@ -646,6 +649,22 @@ export function editTagDescription (tag, description) {
   return {type: EDIT_TAG_DESCRIPTION, payload: {tag, description}}
 }
 
+export function showTagPopover (tagName, slug, position, anchorWidth) {
+  return {type: SHOW_TAG_POPOVER, payload: {tagName, slug, position, anchorWidth}}
+}
+
+export function hideTagPopover () {
+  return {type: HIDE_TAG_POPOVER}
+}
+
+export function fetchTagSummary (tagName, id) {
+  return {
+    type: FETCH_TAG_SUMMARY,
+    payload: {api: true, path: `/noo/community/${id}/tag/${tagName}/summary`},
+    meta: {tagName, id}
```

Comment:
you should use caching for this action so if the mouseover is used more than once it's snappy